fileuploader/file_uploader_util.go

```diff
@@ -0,0 +1,82 @@
+package fileuploader
+
+import (
+	"strings"
+	"time"
+)
+
+type rangePair struct {
+	Start int64
+	End   int64
+}
+
+func stringIsNullOrEmpty(s string) bool {
+	s = strings.TrimSpace(s)
+	if s == "" || len(strings.TrimSpace(s)) == 0 {
```

Comment:
string already had the spaces trimmed in the previous line
```suggestion
	if s == "" || len(s) == 0 {
```